modules/aac-remux/android/src/main/java/expo/modules/aacremux/AacRemuxModule.kt

```
@@ -0,0 +1,145 @@
+        }
+      }
+      try {
+        extractor.release()
+      } catch (_: Exception) {
+      }
+
+      val failure = remuxError ?: cleanupError
+      if (failure != null) {
+        File(dstPath).delete()
+        when (failure) {
+          is CodedException -> throw failure
```

detekt (1.23.8)

[warning] Do not throw an exception within a finally statement. This can discard exceptions and is confusing.

(detekt.exceptions.ThrowingExceptionFromFinally)